src/main/java/org/mitre/synthea/export/Exporter.java

```diff
@@ -491,7 +491,14 @@ private static HealthRecord filterForExport(HealthRecord record, int yearsToKeep
       filterEntries(encounter.conditions, claimItems, cutoffDate, endTime, keepCondition);
 
       // allergies are essentially the same as conditions
-      filterEntries(encounter.allergies, claimItems, cutoffDate, endTime, keepCondition);
+      // But we need to redefine all of the predicates, because we are talking about Allergies as
+      // opposed to Entries... You would think that it would work... but generics are hard
+      Predicate<HealthRecord.Allergy> allergyActive = c -> record.conditionActive(c.type);
```

Comment:
It does just call out to `conditionActive` behind the scenes, but let's use `record.allergyActive` here
```suggestion
      Predicate<HealthRecord.Allergy> allergyActive = c -> record.allergyActive(c.type);
```